ts/Core/DefaultOptions.ts

```diff
@@ -2704,7 +2700,7 @@ const defaultOptions: Options = {
          * @type {Highcharts.ColorString|Highcharts.GradientColorObject|Highcharts.PatternObject}
          */
         backgroundColor: color(palette.neutralColor3)
-            .setOpacity(0.85).get(),
+            .setOpacity(0.85).get() as ColorType,
```

Comment:
We avoid forced types, because it disables some type checking. Why is this necessary, if `Color.get` returns `Color` (part of ColorType)?

Reply:
`.get()` apparently returns `ColorType|undefined`...

Reply:
Maybe we should stop with `undefined` as an input. It does not sound right to me. An alternative for special cases could be something like a `Color.Null` or `Color.NaC` constant instance of `Color`.

Reply:
I think disallowing `undefined` as a color input makes most sense. I'll make a `@todo` comment about it, so we don't double the scope of this PR.